File: August/August 26th/algorithmEx1.js
function arrayMaker(number){
    let arr = []
    let newNum = 0
    let place = 10
    let remainder;
    while(newNum == 0 || number % place != 0) {
        remainder = number % place
        arr.push(remainder)
        place *= 10
        number -= remainder
    }
    return arr
}

let arr2 = array_maker(2798729387492387)
console.log(arr2)

File: August/August 26th/algorithmEx1.py
arr = [1,2, 2, 2, 7,2,3,8,7,2]
arr.sort()
dups = []
for i in range(len(arr)):
    if arr[i - 1] == arr[i] and arr[i-1] not in dups:
        dups.append(arr[i])

print(len(dups))

number = input("Enter a number: ")
arr = []
while(number > 0):
    remainder = number % 10
    number -= remainder
    arr.append(remainder)
    number /= 10
print(arr)